pkg/secrets/pushtofile/secret_group_test.go

```diff
@@ -89,6 +97,24 @@ func goodSecretSpecs() []SecretSpec {
 	}
 }
 
+func failingPushToWriter(
+	writer io.Writer,
+	groupName string,
+	groupTemplate string,
+	groupSecrets []*Secret,
+) error {
+	return errors.New("underlying error message")
+}
+
+func panicPushToWriter(
+	writer io.Writer,
+	groupName string,
+	groupTemplate string,
+	groupSecrets []*Secret,
+) error {
+	panic("canned panic response - maybe containing secrets")
+}
+
```

Comment:
Since each of these functions are being used only once I think it's fine to inline them... to avoid a step of indirection.